Source/PagesController.swift

```diff
@@ -172,6 +174,15 @@ extension PagesController : UIPageViewControllerDelegate {
 
 extension PagesController {
 
+  private func setViewController(viewController: UIViewController, atPage page:Int) {
+    pagesDelegate?.pageViewController(self,
+      setViewController: viewController,
+      atPage: page)
+    if let pageChangeHandler = pageChangeHandler {
```

Comment:
@vadymmarkov Could you go with `pageChangeHandler?(viewController: viewController, page: page)` instead of adding an optional assignment? It looks a little silly when both the target and the assignee has the same name.